biocrnpyler/polymer.py

```diff
@@ -107,44 +143,44 @@ def direction_invert(self,dirname):
             return dirname
 
     def __len__(self):
-        return len(self._polymer)
+        return len(self.polymer)
```

Comment:
an easy potential test line to increase coverage if needed.

Reply:
changed